addon/mixins/translate3d-mixin.js

```diff
@@ -0,0 +1,192 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
```

Comment:
Can this mixin be replaced by https://github.com/peec/ember-css-transitions, @peec?